src/components/SubRequirement.vue

```diff
@@ -82,6 +124,15 @@ export default Vue.extend({
       const type = isCompleted ? 'completed' : 'ongoing';
       this.$emit('toggleDescription', reqIndex, type, subReqIndex);
     },
+    closeMenuIfOpen() {
+      if (this.showFulfillmentOptionsDropdown) {
+        this.showFulfillmentOptionsDropdown = false;
+      }
+    },
```

Comment:
Isn't this just equivalent to
```
closeMenu() {
  this.showFulfillmentOptionsDropdown = false;
}
```
That may be cleaner?

Reply:
oops, right